### ISSUE TITLE
How to use `impute.rfsrc()` in test data with other survival models

### ISSUE THREAD

**Issue by MiqG** [kogalur/randomForestSRC#69]
Hi! 

first, thank you very much for developing such a great package!

After using `randomForestSRC` for a while, I have encountered the problem that it is not clear to me how I can use the imputation forest generated on a training set on the test set for other models.

I found that this can be achieved after fitting a model with `rfsrc()` with the option `na.action = 'na.impute'` combined with `predict.rfsrc()` using `new.data` and again `na.action = 'na.impute'`. 

I would like to repeat the same procedure with a `survival::coxph()` model fitted with a training set imputed with `impute.rfsrc()`, but I do not see how I can reproduce the same "imputing forest" for the test set in this case. And I would like to avoid imputing the whole data set before splitting.

Thank you very much for your help in advance,

Miquel

**Comment by MiqG** [comment]
Thank you very much for the quick reply!

Since there is a function that explicitly allows NA imputation outside from the random forest fitter, I hoped it would be possible to retrieve the underlying imputation model with an option like `forest=TRUE`.

Anyways, I close the issue.
